### DISCUSSION TITLE
ModuleNotFoundError: No module named 'masoniteorm'

### DISCUSSION THREAD

**Discussion by Poraksis** [MasoniteFramework/masonite#535] [Q&A]
I followed the installation at your website but masonite-orm didnt downloaded automatically.I've checked requirements.txt here below.

masonite>=4.0,<5.0
masonite-orm>=2.0,<3.0

then i manually added pip install masonite-orm and problem solved.

**Comment by girardinsamuel** [comment]
When you followed the installation, did you run `project install` inside your project directory after creating the project ?

It's not mentioned in the documentation but it's written in the command line when you create a project.
👉 I will add it to the docs and update the command line to make it more obvious !

Feel free to join our Discord if you have other questions :wink
https://discord.gg/TwKeFahmPZ
